src/Tribe/Main.php

```diff
@@ -358,7 +358,7 @@ protected function addHooks() {
 			add_filter( 'nav_menu_items_' . self::POSTTYPE, array( $this, 'add_events_checkbox_to_menu' ), null, 3 );
 			add_filter( 'wp_nav_menu_objects', array( $this, 'add_current_menu_item_class_to_events' ), null, 2 );
 
-			add_filter( 'generate_rewrite_rules', array( $this, 'filterRewriteRules' ) );
+			add_filter( 'generate_rewrite_rules', array( $this, 'filter_rewrite_rules' ) );
```

Comment:
If you are going to change the method name, we probably should create a deprecated (scheduled for removal in 4.2) method with the same name (`filterRewriteRules`) that just calls `filter_rewrite_rules`. This filter should then have a comment above it stating that it should be swapped out for the new function in 4.2.